packages/office-ui-fabric-react/src/components/Callout/examples/Callout.Status.Example.tsx

```diff
@@ -31,56 +26,32 @@ const styles = mergeStyleSets({
   ],
 });
 
-// Example code
-export class StatusCalloutExample extends React.Component<{}, IStatusCalloutExampleState> {
-  public state: IStatusCalloutExampleState = {
-    isCalloutVisible: false,
-  };
-
-  private _menuButtonElement = React.createRef<HTMLDivElement>();
-
-  public render(): JSX.Element {
-    const { isCalloutVisible } = this.state;
-
-    return (
-      <>
-        <div className={styles.buttonArea} ref={this._menuButtonElement}>
-          <DefaultButton
-            onClick={this._onShowMenuClicked}
-            text={isCalloutVisible ? 'Hide StatusCallout' : 'Show StatusCallout'}
-          />
-        </div>
-        {this.state.isCalloutVisible && (
-          <Callout
-            className={styles.callout}
-            target={this._menuButtonElement.current}
-            onDismiss={this._onCalloutDismiss}
-            role="status"
-            aria-live="assertive"
-          >
-            <DelayedRender>
-              <>
-                <p className={styles.subtext}>
-                  This message is treated as an aria-live assertive status message, and will be read by a screen reader
-                  regardless of focus.
-                </p>
-              </>
-            </DelayedRender>
-          </Callout>
-        )}
-      </>
-    );
-  }
-
-  private _onShowMenuClicked = (): void => {
-    this.setState({
-      isCalloutVisible: !this.state.isCalloutVisible,
-    });
-  };
-
-  private _onCalloutDismiss = (): void => {
-    this.setState({
-      isCalloutVisible: false,
-    });
-  };
-}
+export const CalloutStatusExample: React.FunctionComponent = () => {
```

Comment:
I think maybe this should be 
```suggestion
export const StatusCalloutExample: React.FunctionComponent = () => {
```
From what I can tell this might be why the build is failing. @czearing Thanks a bunch for submitting this change!

Reply:
Oops that seems like it could be the issue. Thanks for reviewing @joschect!